### DISCUSSION TITLE
Multiple styles

### DISCUSSION THREAD

**Discussion by drifteaur** [openmaptiles/openmaptiles#1692] [Q&A]
What would I need to do if I wanted to support multiple styles for rendering? 

I can see I can add more style options in `style/config.json` - but I'm not sure how to include those in the build process.

**Comment by nikolaydyankov** [comment]
I'm not sure if this is the correct way, but it worked for me:

1. Add your new style in style/config.json in the `styles` prop
2. Put the new my-style.json in build/style
3. In my-style.json replace sources, sprite and glyphs with values from build/style/style.json
4. Restart server
